app/src/App.vue

```diff
@@ -46,7 +40,7 @@ const {
 } = useTipsBalance()
 
 const userStore = useUserDataStore()
-const { name, address } = storeToRefs(userStore)
+const { name, address, isAuth } = storeToRefs(userStore)
```

Comment:
The issue with this approach is that getters and setters do not provide reactivity and using storeToRef will ensure that state changes are reactive and can be easily tracked by vue.
And also I think it would increase too many lines of code unnecessarily

Reply:
The variable userStore.isAuth is reactive and let you know the origin of the value you are using.
The getter are not necessary if we can use it simply like this userStore.isAuth for ex.

Reply:
Got it thanks!